packages/model-serving/src/components/deploymentWizard/useDeploymentWizard.ts

```diff
@@ -1,16 +1,26 @@
+import React from 'react';
+import { useHardwareProfileConfig } from '@odh-dashboard/internal/concepts/hardwareProfiles/useHardwareProfileConfig';
+import { useParams } from 'react-router-dom';
 import { K8sNameDescriptionFieldData } from '@odh-dashboard/internal/concepts/k8s/K8sNameDescriptionField/types';
 import { useK8sNameDescriptionFieldData } from '@odh-dashboard/internal/concepts/k8s/K8sNameDescriptionField/K8sNameDescriptionField';
 import { extractK8sNameDescriptionFieldData } from '@odh-dashboard/internal/concepts/k8s/K8sNameDescriptionField/utils';
-import { useHardwareProfileConfig } from '@odh-dashboard/internal/concepts/hardwareProfiles/useHardwareProfileConfig';
 import type { SupportedModelFormats } from '@odh-dashboard/internal/k8sTypes';
-import { useModelTypeField, type ModelTypeFieldData } from './fields/ModelTypeSelectField';
+import { byName, ProjectsContext } from '@odh-dashboard/internal/concepts/projects/ProjectsContext';
+import { LabeledConnection } from '@odh-dashboard/internal/pages/modelServing/screens/types';
 import { useModelFormatField } from './fields/ModelFormatField';
+import { ModelLocationData } from './fields/modelLocationFields/types';
+import { useModelTypeField, type ModelTypeFieldData } from './fields/ModelTypeSelectField';
+import { useModelLocationData } from './fields/ModelLocationInputFields';
 
 export type ModelDeploymentWizardData = {
   modelTypeField?: ModelTypeFieldData;
   k8sNameDesc?: K8sNameDescriptionFieldData;
   hardwareProfile?: Parameters<typeof useHardwareProfileConfig>;
   modelFormat?: SupportedModelFormats;
+  modelLocationData?: ModelLocationData;
+  connections?: LabeledConnection[];
+  initSelectedConnection?: LabeledConnection | undefined;
```

Comment:
I think we should move these out of the wizard data since they are dependencies for the form data but not necessarily what we need to submit the deployment